ui/imports/shared/controls/AmountInput.qml

```diff
@@ -10,14 +10,15 @@ import utils 1.0
 Input {
     id: root
 
-    property int maximumLength: 10
+    property int maximumLength: 0
```

Comment:
Does it make sense to keep this property if we will not have a "maximum amount"??

Reply:
I'm not sure it will be needed in the future. That's why I left it optional. I can remove this property

Reply:
fixed ✅